src/dispatch/incident/flows.py

```diff
@@ -630,25 +630,22 @@ def incident_create_flow(*, incident_id: int, checkpoint: str = None, db_session
     )
 
 
-@background_task
-def incident_active_flow(incident_id: int, command: Optional[dict] = None, db_session=None):
+def incident_active_status_flow(incident: Incident, db_session=None):
     """Runs the incident active flow."""
-    # we load the incident instance
-    incident = incident_service.get(db_session=db_session, incident_id=incident_id)
-
-    # we remind the incident commander to write a tactical report
-    send_incident_report_reminder(incident, ReportTypes.tactical_report, db_session)
+    # we un-archive the conversation
+    convo_plugin = plugin_service.get_active(db_session=db_session, plugin_type="conversation")
+    if convo_plugin:
+        convo_plugin.instance.unarchive(incident.conversation.channel_id)
```

Comment:
Should we only do this when we go from closed -> active? Slack will throw a `not_archived` error if the channel is not archived: https://api.slack.com/methods/conversations.unarchive#errors. Or is the plan to ignore the error?

Reply:
I think we can handle this error in the slack plugin, i.e. if we ask for it to be unarchived and it's already unarchived we should swallow that exception.